plugins/UpdateChecker/UpdateChecker.py

```diff
@@ -78,6 +78,14 @@ def _extractVersionAndURLFromData(cls, data: Dict, application_name: str) -> Tup
         if os not in data[application_name]:
             return None, None
 
+        if "postfix_type" in data[application_name][os] and "postfix_version" in data[application_name][os]:
+            # Prerelease versions include the extra postfix_type and postfix_version keys
+            return Version([int(data[application_name][os]["major"]),
```

Comment:
I would feel more saver when we apply a bit of defensive programming here, with a try-catch block guarding against weird input